# Type: fix
# Subtype: Ix
# English Summary: Modified color settings
# Chinese Summary: 修改颜色设置
# Entry Number: 9

# === Git Diff Content ===

diff --git a/OsmAnd/src/net/osmand/plus/views/MapControlsLayer.java b/OsmAnd/src/net/osmand/plus/views/MapControlsLayer.java
index c8081d39a5..ad9221e45b 100644
--- a/OsmAnd/src/net/osmand/plus/views/MapControlsLayer.java
+++ b/OsmAnd/src/net/osmand/plus/views/MapControlsLayer.java
@@ -130,12 +130,12 @@ public class MapControlsLayer extends OsmandMapLayer {
 		}
 		int sh = Color.WHITE;
 		if (nightMode != null && nightMode.isNightMode()) {
-			sh = 0xffC8C8C8;
+			sh = Color.TRANSPARENT;
 		}
 		if(shadowColor != sh) {
 			shadowColor = sh;
-			zoomTextPaint.setColor(sh == Color.WHITE ? Color.BLACK : sh);
-			rulerTextPaint.setColor(sh == Color.WHITE ? Color.BLACK : sh);
+			zoomTextPaint.setColor(sh == Color.WHITE ? Color.BLACK : 0xffC8C8C8);
+			rulerTextPaint.setColor(sh == Color.WHITE ? Color.BLACK : 0xffC8C8C8);
 		}
 		if (showZoomLevel || !view.getSettings().SHOW_RULER.get()) {
 			drawZoomLevel(canvas);
